internal/database/metadata/mysql/schema.go

```diff
@@ -55,20 +55,13 @@ var META_TABLE_SCHEMAS = map[string]string{
 		`,
 }
 
-var META_TABLE_FOREIGN_KEYS = map[string]string{
-	"feature": `
-		ALTER TABLE feature
-		ADD FOREIGN KEY (group_id) REFERENCES feature_group(id)
-	`,
-	"feature_group": `
-		ALTER TABLE feature_group
-		ADD FOREIGN KEY (entity_id) REFERENCES entity(id),
-		ADD FOREIGN KEY (online_revision_id) REFERENCES feature_group_revision(id)
-	`,
-	"feature_group_revision": `
-		ALTER TABLE feature_group_revision
-		ADD FOREIGN KEY (group_id) REFERENCES feature_group(id)
-	`,
+var META_TABLE_FOREIGN_KEYS = []string{
```

Comment:
it doesn't need to be a `map`. Shall we modify other metadata schemas as well?

Reply:
Using `slice` does accomplish the same thing, but `map` makes the code clearer and more readable.

Reply:
We can't use a map here since there are 2 constraints for a same table

Reply:
o-o there `slice` is fine.